gui/datumaro_gui/components/single/tabs/export.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import os.path as osp
+
+import streamlit as st
+from datumaro_gui.utils.dataset.data_loader import DataRepo, SingleDatasetHelper
+from datumaro_gui.utils.dataset.state import get_download_folder_path
+from streamlit import session_state as state
+
+
+def main():
+    tasks = ["classification", "detection", "instance_segmentation", "segmentation", "landmark"]
```

Comment:
It seems that multiple and single export use same code a lot. 
I think this situation is similar in other tabs.
How about refactoring this later? Please make a JIRA Task for this purpose.

Reply:
I will update this in following tasks. Thank you for letting me know this.